src/lib/edit_script.rs

```diff
@@ -178,7 +185,8 @@ impl Chawathe96Config {
                                                  to_in_order: &HashSet<NodeId<ToNodeId>>)
                                                  -> u16 {
         // 1. Let y=p(x) in T_2 and let w be the partner of x in T_1.
-        // N.B. w seems to be unused in the algorithm.
+        // N.B. w seems to be unused in the algorithm and y is not needed in
```

Comment:
So I have to ask: why is `w` defined, do you think? Is it an accident or ...?

Reply:
I think it's a typo. `w` is used in the other two functions / methods so I would guess it's a copy & paste error.

Reply:
Is it a parameter to the function or ... ?

Reply:
No, it's just a variable. Most of the comments in this code are lifted directly from the algorithm in the paper, which says `1. Let y=p(x) in T_2 and let w be the partner of x (x in T_1).`. In this notation `p(x)` means "parent of `x`". The other steps do not refer to `w` at all.

Reply:
So do you think `w` is a typo for another variable or is the whole "let `w` be the partrner of x" bit redundant?

Reply:
I think that phrase is redundant, all the other variables are clearly defined and seem perfectly sensible!

Reply:
OK, so could you expand the comment to make this clearer for worriers like me? :)